Source/WebCore/Modules/WebGPU/Implementation/WebGPUBufferImpl.cpp

```diff
@@ -89,7 +89,7 @@ auto BufferImpl::getBufferContents() -> MappedRange
         return { nullptr, 0 };
 
     auto* pointer = wgpuBufferGetBufferContents(m_backing.get());
-    auto bufferSize = wgpuBufferGetSize(m_backing.get());
+    auto bufferSize = wgpuBufferGetActualSize(m_backing.get());
```

Comment:
Would it be clearer if these were called `wgpuBufferGetInitialSize` and `wgpuBufferGetCurrentSize`?